#Details of service data types
bool motion
---
bool response
